util/CMakeLists.txt

```
@@ -245,8 +245,10 @@ if(ANDROID)
 endif()
 
 if(LINUX OR ANDROID)
+    find_package(CURL REQUIRED)
```

I think here it would be best to mimic the configuration of [upstream](https://chromium.googlesource.com/crashpad/crashpad/+/refs/heads/main/util/net/tls.gni#18):

```suggestion
    if (LINUX)
        find_package(CURL REQUIRED)
        target_include_directories(crashpad_util PRIVATE ${CURL_INCLUDE_DIRS})
        target_link_libraries(crashpad_util PRIVATE ${CURL_LIBRARIES})
        SET(HTTP_TRANSPORT_IMPL net/http_transport_libcurl.cc)
    else()
        SET(HTTP_TRANSPORT_IMPL net/http_transport_socket.cc)
    endif()
```